controllers/controllers/cluster_controller.go

```diff
@@ -18,6 +24,11 @@ import (
 	"github.com/aws/eks-anywhere/pkg/providers/vsphere"
 )
 
+const (
+	defaultRequeueTime          = time.Minute
+	clusterFinalizerName string = "clusters.anywhere.eks.amazonaws.com/finalizer"
```

Comment:
is there a specific reason to include "string" type here?

Reply:
no. removed